tests/test_mod.py

```diff
@@ -11,6 +11,23 @@
 from sentinelsat.sentinel import (InvalidChecksumError, SentinelAPI, SentinelAPIError, convert_timestamp, format_date,
                                   get_coordinates, md5_compare)
 
+_small_query = dict(
+        area='0 0,1 1,0 1,0 0',
+        initial_date=datetime(2015, 1, 1),
+        end_date=datetime(2015, 1, 2))
+
+_large_query = dict(
+        area='0 0,0 10,10 10,10 0,0 0',
+        initial_date=datetime(2015, 1, 1),
+        end_date=datetime(2015, 12, 31))
+
+_api_auth = dict(
```

Comment:
instead of `environ` I suggest to get the credentials with `environ.get`. This way we don't get an exception on TravisCI when a PR is created from a fork.